### DISCUSSION TITLE
Unreleased 1.6.0.0-PN-SNAPSHOT

### DISCUSSION THREAD

**Discussion by Polyana-Fontes** [PowerNukkit/PowerNukkit#1260] [Show and tell]
## [Unreleased 1.6.0.0-PN] - Future ([Check the milestone](https://github.com/PowerNukkit/PowerNukkit/milestone/29?closed=1))
Click the link above to see the future.

This work in progress version supports Minecraft `1.18.0`.

### Changed
- #1258 Changed supported version to Minecraft Bedrock Edition `1.18.0`.

# Download
Download the last snapshot at https://powernukkit.org/snapshot
Choose the one that ends with `-shaded.jar` (probably: `powernukkit-1.6.0.0-PN-SNAPSHOT-shaded.jar`)

[Unreleased 1.6.0.0-PN]: https://github.com/PowerNukkit/PowerNukkit/compare/v1.5.2.0-PN...bleeding

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerNukkit/PowerNukkit/releases/tag/v1.6.0.0-PN-SNAPSHOT-2021.12.01'>Unreleased 1.6.0.0-PN-SNAPSHOT</a>.</em>

**Comment by ReiyansCN** [comment]
Sir，can you prepare a Chinese payment method such as Alipay at your donation office? In fact, we would like to donate to you, but the bank card and paypal I have been unable to pay normally …
